Attendance_app.py

```diff
@@ -40,8 +40,7 @@ def __init__(self, *args, **kwargs):
 
         self.title_font = tkfont.Font(family='Helvetica', size=18, weight="bold", slant="italic")
         x, y = pyautogui.size()
-        geo = str(int(0.35*x))+'x'+str(int(0.85*y))
-        self.geometry(geo)
+        self.geometry("700x550")

```

Comment:
Converting dynamic window size to static would result in apparent different app window size for different screen sizes.
Consider reverting back to dynamic window size.